forte/base_classes/active_space_solver.cc

```diff
@@ -826,4 +827,13 @@ compute_average_state_energy(const std::map<StateInfo, std::vector<double>>& sta
     return average_energy;
 }
 
+CICouplingCoefficients ActiveSpaceSolver::coupling_coefficients(const StateInfo& state,
+                                                                int max_level) {
+    return state_method_map_[state]->coupling_coefficients(max_level);
```

Comment:
I'd prefer the use of `at` in place of square brackets. Sadly, we cannot guarantee that `state` will actually be in `state_method_map_`.

Reply:
Changed to `at` and marked as `const`.